tests/test_ingest_chatgpt_linux.py

```
@@ -0,0 +1,1979 @@
+                "--repo-dir",
+                "/var/lib/arch-pkgs-ingest-interruption-test",
+                "--repo-name",
+                "fixture",
+            ],
+            cwd=REPO_ROOT,
+            env=environment,
+            text=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            check=False,
+        )
```

Ruff (0.16.1)

[warning] Prefer `capture_output` over sending `stdout` and `stderr` to `PIPE`

Replace with `capture_output` keyword argument

(UP022)